spec/mailers/comment_mailer_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+ 
+describe CommentMailer do
+  describe "#notification_new_comment" do
+    
+    let!(:comment) { FactoryGirl.build(:comment) }       
+    let(:mail) { CommentMailer.notification_new_comment(comment.parent.user.id, nil)}
+    #binding.pry
+    it 'renders the subject' do
+      expect(mail.subject).to eql('New comment from your post')
+    end
+ 
+    it 'renders the receiver email' do
+       expect(mail.to).to eql([comment.parent.user.email])
+    end
+ 
+    it 'renders the sender email' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.